src/scripts/create_machine.sh

```diff
@@ -191,7 +191,7 @@ fi
 
 if [ "$START_VM" = "true" ]; then
     vmExists "$VM_NAME" || handleFailure $?
-    virsh -c "$CONNECTION_URI" -q dumpxml "$VM_NAME" > "$XMLS_FILE"
+    virsh -c "$CONNECTION_URI" -q dumpxml --inactive "$VM_NAME" > "$XMLS_FILE"
```

Comment:
Whee, I wasn't aware of `--inactive` either -- this totally makes sense (and it even feels like that would be a better default..)